src/util/Validation.ts

```diff
@@ -0,0 +1,21 @@
+import { Severity } from "../model/form/ValidationResult";
+
+/**
+ * Constants and utility functions concerning server-side quality validation.
+ */
+const Validation = {
+  QUALITY_AFFECTING_RULES: [
+    "https://slovník.gov.cz/jazyk/obecný/g4",
+    "https://slovník.gov.cz/jazyk/obecný/m1",
+    "https://slovník.gov.cz/jazyk/obecný/g13",
+    "https://slovník.gov.cz/jazyk/obecný/g14",
+  ],
+
+  toSeverity: function (ruleId?: string) {
+    return ruleId && this.QUALITY_AFFECTING_RULES.indexOf(ruleId) !== -1
```

Comment:
Cannot see where this manifestates, but I thought that QUALITY_AFFECTING_RULES should end-up with warning, not error.

Reply:
The severities are
* BLOCKER
* ERROR
* WARNING
Where BLOCKER prevents form from being submitted. ERROR is for violation of rules affecting term quality, the remainder are WARNING. Of course, the enum constants can be adjusted if need be.